src/Auth0.AuthenticationApi/AuthenticationApiClient.cs

```diff
@@ -495,6 +495,50 @@ public Task<PushedAuthorizationRequestResponse> PushedAuthorizationRequestAsync(
             );
         }
 
+        /// <inheritdoc/>
+        public Task DeleteMfaAuthenticatorAsync(
+            DeleteMfaAuthenticatorRequest request,
+            CancellationToken cancellationToken = default)
+        {
+            if (request == null)
+            {
+                throw new ArgumentNullException(nameof(request));
+            }
+
+            if (string.IsNullOrEmpty(request.AccessToken))
+            {
+                throw new InvalidOperationException($"{nameof(request.AccessToken)} is required");
+            }
+
+            if (string.IsNullOrEmpty(request.AuthenticatorId))
+            {
+                throw new InvalidOperationException($"{nameof(request.AuthenticatorId)} is required");
+            }
```

Comment:
We do not need to introduce this kind of validation here.Currently, we rely on the backend to tell us if something is missing, mainly because it isn't always clear what is and isnt required.

Reply:
removed this